src/clusterer.rs

```diff
@@ -166,6 +158,45 @@ fn partition_sketches(
 //     return to_return;
 // }
 
+pub struct FastaniClusterer {
```

Comment:
@wwood Should I move this to fastani.rs?

Reply:
sure